src/app.ts

```diff
@@ -153,4 +154,30 @@ export class SpaceMeshApp extends BaseApp {
             throw processErrorResponse(e)
         }
     }
+
+    async signMessage(path: BIP32Path, signMessage: MsgSigner): Promise<ResponseSign> {
+        const bs = new ByteStream()
+        bs.appendUint16(signMessage.prefix.length)
+        bs.appendUint16(signMessage.message.length)
+        bs.appendBytes(signMessage.prefix)
+        bs.appendUint8(signMessage.domain)
+        bs.appendBytes(signMessage.message)
+
+        const payload = bs.getCompleteBuffer()
+
+        const chunks = this.prepareChunks(path, payload)
+        try {
+            let signatureResponse = await this.signSendChunk(this.INS.SIGN_MESSAGE, 1, chunks.length, chunks[0])
```

Comment:
Is this one starting from 1? I am not 100% into the signSendChunk function.

Reply:
Yes, this have to start in 1